src/data/polynomial/degree/lemmas.lean

```diff
@@ -239,6 +264,30 @@ end
 end degree
 end semiring
 
+section ring
+
+variables [ring R] {p q : R[X]}
+
+lemma nat_degree_sub_le_iff_left (qn : q.nat_degree ≤ n) :
```

Comment:
How about also adding the "right" version of this lemma:
```lean
lemma nat_degree_sub_le_iff_right (pn : p.nat_degree ≤ n) :
  (p - q).nat_degree ≤ n ↔ q.nat_degree ≤ n :=
```

Reply:
Done!  I also added the "symmetric" lemma

```lean
lemma nat_degree_sub : (p - q).nat_degree = (q - p).nat_degree  :=
```